mautic/mautic
# [UX] Add a link to campaign creation in the Dashboard

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The latest update to the Dashboard introduces a direct link for creating new campaigns, streamlining the user workflow. This addition makes it more convenient for users to initiate campaign creation without navigating away from the Dashboard. It’s a quality-of-life improvement that benefits users who frequently create campaigns, enhancing their productivity and simplifying the process.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Dashboard
3. Look at the new button

![image](https://github.com/mautic/mautic/assets/116097999/ae7ecc27-2b54-45fd-a60a-f24af2e64626)
